src/components/modal/style.ts

```diff
@@ -7,9 +7,16 @@ export const ModalContainer = styled.div<ModalOpen>`
   justify-content: center;
   align-items: center;
   background: rgba(0, 0, 0, 0.7);
-  position: absolute;
+  /* position: absolute;
   z-index: ${({ open }) => (open ? "2" : "-1")};
   opacity: ${({ open }) => (open ? "1" : "0")};
   width: ${({ open }) => (open ? "100%" : "0")};
-  height: ${({ open }) => (open ? "100%" : "0")};
+  height: ${({ open }) => (open ? "100%" : "0")}; */
```

Comment:
This is an anti-pattern. We cannot manipulate the width / height property in css. There is opacity and visibility for this to hide and unhide page elements.

Reply:
simply delete code instead of commenting it out